assets/src/components/v2/elevator/current_elevator_closed.tsx

```diff
@@ -67,7 +67,9 @@ const CurrentElevatorClosed = ({
           <div className="subheading">Accessible Path</div>
           <div>
             <IsaNegative width={100} height={100} />
+            {accessiblePathDirectionArrow} ? (
             <Arrow direction={accessiblePathDirectionArrow} className="arrow" />
```

Comment:
I think we should update

```ts
accessible_path_direction_arrow: Direction;
```

in the type definition above to instead be

```ts
accessible_path_direction_arrow: Direction | null;
```

to align with the reality of the API. That's ultimately why TS didn't raise an error on this line, forcing us to handle the possible `null`.

Reply:
good callout! updated with this